android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -361,11 +360,49 @@ String getNextTrackUUID() {
 
         do {
             uuid = UUID.randomUUID().toString();
-        } while (mMediaStreamTracks.containsKey(uuid));
+        } while (getTrackForId(uuid) != null);
 
         return uuid;
     }
 
+    MediaStream getStreamForReactTag(String streamReactTag) {
+        MediaStream stream = localMediaStreams.get(streamReactTag);
+
+        if (stream != null) {
+            return stream;
+        }
+
+        for (int idx = 0; idx < mPeerConnectionObservers.size(); idx++) {
+            PeerConnectionObserver pco
+                = mPeerConnectionObservers.valueAt(idx);
+            stream = pco.remoteMediaStreams.get(streamReactTag);
+            if (stream != null) {
+                return stream;
+            }
+        }
+
+        return null;
+    }
+
+    MediaStreamTrack getTrackForId(String trackId) {
+        MediaStreamTrack track = localMediaStreamTracks.get(trackId);
+
+        if (track != null) {
+            return track;
+        }
+
+        for (int idx = 0; idx < mPeerConnectionObservers.size(); idx++) {
```

Comment:
nope, stupid me was installing webrtc-react-native from Github instead of local dev - not possible